vintage/citrus-java-dsl/src/main/java/com/consol/citrus/dsl/builder/HttpActionBuilder.java

```diff
@@ -26,7 +27,7 @@ public HttpClientActionBuilder client(String httpClient) {
         return new HttpClientActionBuilder(delegate.client(httpClient));
     }
 
-    public HttpServerActionBuilder server(HttpServer httpServer) {
+    public HttpServerActionBuilder server(Endpoint httpServer) {
```

Comment:
thanks @tschlat! that's the right one - follow up for #817. my bad.

Reply:
yes, that's the right place. 👍 

@bbortt what about reverting the changes made in #817? I think the builder should only accept HttpServer endpoints. I need to review the use case in the citrus-simulator to understand why another endpoint type is required here.

It is ok to provide the Endpoint option in the vintage modules but I think using the builder with generic endpoint type is not the way we should go in the long run.

Reply:
see https://github.com/citrusframework/citrus/pull/840.